generators/utils.js

```diff
@@ -667,12 +669,40 @@ function loadBlueprintsFromConfiguration(config) {
  * @returns {Array} an array that contains the info for each blueprint
  */
 function parseBluePrints(blueprints) {
-    if (blueprints && !Array.isArray(blueprints)) {
+    if (Array.isArray(blueprints)) {
+        return blueprints;
+    }
+    if (typeof blueprints === 'string') {
         return blueprints
             .split(',')
             .filter(el => el != null && el.length > 0)
             .map(blueprint => parseBlueprintInfo(blueprint));
     }
+    return [];
+}
+
+function mergeBlueprints(...blueprintsToMerge) {
```

Comment:
Judging by the tests, this function does two things:
  1. flattens the internal structure of the array `([ [{}], [{}] ] => [ {}, {} ])`
  2. it resolves versions based on the input while removing duplicates

It's not clear what this function does by just looking at the code (the tests helped me)

You can do all that by:
  - flattening the array (Array#flat)
  - creating a map for (2), filling the map by using your algorithm and then getting the map's values

Reply:
Array.flat requires node 12, so I did the bump.
Maps doesn't have order guarantee.

Reply:
Maps have order guarantee :)
From the MDN doc:
The Map object holds key-value pairs and remembers the original insertion order of the keys.

Reply:
Yep, only for english js, not for portuguese js... 🤕 
Order is not mentioned at portuguese translation.